WordPress/Classes/ViewRelated/New Group/SiteCreationOutput.swift

```diff
@@ -0,0 +1,3 @@
+// The final output of the site creation wizard, is an endocable model object that can be submitted to the backend
+struct SiteCreationOutput {
```

Comment:
The existing implementation uses a struct named `SiteCreationParams` to interface with `SiteCreationService`. There might be opportunities for reuse (or extension) of the service, but that's outside the scope of the PR. I suppose I'm calling it out so we can think about it as we move forward.